config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-upgrade.yaml

```diff
@@ -94,34 +94,3 @@ periodics:
         requests:
           memory: "9000Mi"
           cpu: 2000m
-- name: ci-kubernetes-e2e-kubeadm-kinder-upgrade-1-11-1-12
```

Comment:
this job should not be removed. it upgrades a unsupported version to a supported version.

Reply:
Thanks will revert.